phpBB/adm/style/acp_profile.html

```diff
@@ -17,7 +17,7 @@
 		</div>
 	<!-- ENDIF -->
 
-	<form id="add_profile_field" method="post" action="{U_ACTION}">
+	<form id="add_profile_field" method="post" action="{U_ACTION}"{S_FORM_ENCTYPE}>
```

Comment:
I want to use the following code in the extension or wherever:
$template->assign_var('S_FORM_ENCTYPE', ' enctype="multipart/form-data"');
And use this exactly the same way in all places where it is needed:
- ucp_profile_profile_info.html (already existing, and coded as I want it)
- acp_profile.html 
- acp_users_profile.html
- ucp_register.html
If I changed to what you propose, that would mean that the usage pattern would be different in each place, and the template var should have a different value depending on where it is used.  Not the way I am coding this.
Besides, if you asked me to change the already existing var to follow the "new" pattern, that would break any existing code and extensions that make use of the existing var.
So, I stand by my proposal.  I want to keep it as I originally proposed.